robottelo/vm.py

```diff
@@ -56,13 +51,22 @@ def __init__(
             self, cpu=1, ram=512, distro=None, provisioning_server=None,
             image_dir=None, tag=None, hostname=None, domain=None,
             target_image=None):
+        distro_el6 = settings.distro.image_el6
+        distro_el7 = settings.distro.image_el7
+        base_images = [distro_el6, distro_el7]
         self.cpu = cpu
         self.ram = ram
-        self.distro = BASE_IMAGES[-1] if distro is None else distro
-        if self.distro not in BASE_IMAGES:
+        if distro == 'rhel6':
+            distro = distro_el6
+        if distro == 'rhel7':
+            distro = distro_el7
+        if distro is None:
+            distro = distro_el7
+        self.distro = distro
```

Comment:
wth is this? I dont get why do we have it in constants and overriding here with settings

if distro == 'rhel6' then set it 'rhel68' from settings? can we drop those constants? two layers for one stuff